scripts/sync_codemirror.sh

```diff
@@ -44,11 +42,17 @@ excluded_dirs=".github .circleci"
 # Go to the root of the repo
 cd "$(git rev-parse --show-cdup)" || exit 1
 
-source_dir="$(pwd)/${source_path}"
-
 tmp_dir="$(mktemp -d)"
 trap 'rm -rf "${tmp_dir}"' EXIT
 
+source_dir() {
+  source_path="$1"
+  cd "$(git rev-parse --show-cdup)" || exit 1
```

Comment:
What does this do? i do not understand. can we use a parameter like `pwd` instead?

Reply:
well I used the same way than here: https://github.com/prometheus/prometheus/pull/9652/files#diff-85009413deae27e1f38c6fd2251b45d431ed591841fe1fd91b78f30618e81ed1R43

So maybe we can indeed simply it with just `pwd`. But I guess there was a good idea on why we decide to use this "hack" than using a simple `pwd`.